test/manifests/normal/scaleup.yaml

```diff
@@ -1,3 +1,4 @@
+location: "{{ .Env.AZURE_REGION }}"
```

Comment:
https://github.com/openshift/openshift-azure/issues/879

Reply:
just ran into this :)

Reply:
Note that scale up with a partial request is still not going to work in prod, see https://github.com/openshift/openshift-azure/issues/876.